guides/common/modules/proc_filtering-and-removing-facts.adoc

```diff
@@ -0,0 +1,53 @@
+////
+Base the file name and the ID on the module title. For example:
+* file name: proc-doing-procedure-a.adoc
+* ID: [id="proc-doing-procedure-a_{context}"]
+* Title: = Doing procedure A
+
+The ID is an anchor that links to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
+
+The `context` attribute enables module reuse. Every module ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
+////
+
+////
+Indicate the module type in one of the following
+ways:
+Add the prefix proc- or proc_ to the file name.
+Add the following attribute before the module ID:
+////
+:_content-type: PROCEDURE
+
+[id="proc_filtering-and-removing-facts_{context}"]
+= Optimizing performance by removing NICs from database
```

Comment:
title, anchor, and file name do not match. Please resolve this & also remove all content above line 20.